apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -168,89 +166,60 @@ class NewRewardClass extends React.Component {
               Math.floor(disbursement) === +disbursement
         )
       ) &&
-        semanticErrors.length === 0
+        errors.length === 0
     )
     return valid
   }
 
-  setSemanticErrors = (changed) => {
+  // FIXME: What exactly is this doing? What's a good name for it?
```

Comment:
@topocount do you know the answer to this question?

Reply:
I think `getReferenceToken` would be a better name